README.md

```diff
@@ -25,8 +25,8 @@ Install by running ``sudo apt install libgtk-3-0 libnss3``
 
 ## Development
 Recommended setup for development:
-1. Open a command prompt and run ``npm run build`` and wait for it to finish
-2. In the same command prompt as before, run ``npm run watch`` and let the prompt stay open
+1. Clone the repository with ``git clone --recurse-submodules https://github.com/FlashpointProject/launcher.git launcher``
```

Comment:
This section of the README is about how to build the launcher and how to run that build.
This change should instead be made to the ``Setup`` section above, which is about how to download the project and install the dependencies.

Reply:
I genuinely have no idea how I ended up putting it there.